.github/workflows/main.yml

```diff
@@ -77,18 +77,16 @@ jobs:
       - name: Toolchain setup
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2022-01-14
+          toolchain: nightly-2022-05-06
```

Comment:
I originally tried to use 1.60.0 here, but ibcontainer enables the unstable feature when coverage is enabled, so I reverted to nightly.

https://github.com/containers/youki/blob/1591680993aa9c63cbc223e111af7c130aa319fe/crates/libcontainer/src/lib.rs#L1

Reply:
After more looking into it, I found that the actual usage of no_coverage is only here, and it is no-op (because applied to `use` statement).

https://github.com/containers/youki/blob/e0843af2032f9e4d8aeefd87f8022feb5394e943/crates/libcontainer/src/syscall/linux.rs#L2-L3

So I believe we can remove the use of the unstable feature.

Reply:
Removed the use of the unstable feature and changed to use rust 1.60.0: https://github.com/containers/youki/pull/898/commits/a7e800fb819b3d1f12b54fa0fe939cb8b9174d8e